tests/clientTests/componentsTests/button.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Button from '../../../client/components/button';
+
+describe('Button container', () => {
+  const button = mount(<Button id="456" restaurant={123} score={4} type="Cool" />);
+
+  test('Button should be stateful', () => {
+    expect(button).toHaveProperty('state');
```

Comment:
This is testing whether button has a property of state? That doesn't seem to be a property on button from what I see here.

Reply:
It's just a test to make sure I made buttons stateful (using this.state) and not a regular component. Will be more important later when I add button functionality.